airbyte-workers/src/main/java/io/airbyte/workers/config/WorkerConfigurationBeanFactory.java

```diff
@@ -85,6 +85,18 @@ public Map<String, String> specJobKubeAnnotations(@Value("${airbyte.worker.spec.
     return splitKVPairsFromEnvString(kubeAnnotations);
   }
 
+  @Singleton
+  @Named("customNodeSelectors")
```

Comment:
decide to use `useIsolatedNodeSelector` to keep align with other names (such as specJobKubeNodeSelectors). Since this is to define the actual kube selector I think we should just include `node` in the name